mmaction/datasets/tube_dataset.py

```diff
@@ -0,0 +1,183 @@
+import copy
+import os.path as osp
+import pickle
+from collections import defaultdict
+
+from .base import BaseDataset
+from .registry import DATASETS
+
+
+@DATASETS.register_module()
+class TubeDataset(BaseDataset):
```

Comment:
Is `evaluate` needed to be overridden to apply metrics such as map or map_with_error?